### DISCUSSION TITLE
DPI Change via Key Press

### DISCUSSION THREAD

**Discussion by QuentinDOS** [sezanzeb/input-remapper#237] [General]
I was hoping to check if it was possible to change the mouse speed while a button was pressed or held. Like a Sniper mode, my mouse has a button to be used for that but I don't have a way to configure it to do such and was hoping to see if there was a way or if one could be added. Any help would be great, thank you.

**Comment by sezanzeb** [comment]
See https://github.com/sezanzeb/key-mapper/issues/99

**Reply by QuentinDOS** [reply]
Thank you! I greatly appreciate the reply, this software has been really helpful. I use a mouse that's none gaming (which limits my options) and has six buttons and this software so far has been nearly the only one that's allowed me to make use of that button on Linux. I look forward to seeing how this develops. It's been really nice to have.
